handsontable/src/selection/selection.js

```diff
@@ -628,13 +634,16 @@ class Selection {
    * `false` otherwise.
    * @param {boolean} [includeColumnHeaders=false] `true` If the selection should include the column
    * headers, `false` otherwise.
-   * @param {{row: number, col: number}} [focusPosition] The argument allows changing the cell/header
+   * @param {object} [options] Additional object with options.
+   * @param {{row: number, col: number}} [options.focusPosition] The argument allows changing the cell/header
```

Comment:
Haven't we decided that newly created API should use `column` instead of `col`?

Reply:
Not for coordinates. Those props should be compatible with CellCoords and CellRange classes.